infra/grafana/config/grafana.ini

```
@@ -12,4 +12,7 @@ enabled = true
 default_theme = dark
 
 [feature_toggles]
-enable = tempoServiceGraph tempoSearch tempoBackendSearch traceToMetrics traceqlEditor
+enable = tempoServiceGraph tempoSearch tempoBackendSearch traceToMetrics traceqlEditor topNav
```

All lowercase for `topnav` to get it enabled 

```suggestion
enable = tempoServiceGraph tempoSearch tempoBackendSearch traceToMetrics traceqlEditor topnav
```